docs/man/docker-create.1.md

```diff
@@ -66,6 +67,12 @@ IMAGE [COMMAND] [ARG...]
 **--cpuset**=""
    CPUs in which to allow execution (0-3, 0,1)
 
+**--dev**=host
+   Use the systems /dev inside of the container
+     **host**: use the host's /dev directory inside the container.
+     No devices will be mounted on the /dev or any of its subdirs, from inside of the container
```

Comment:
Does this mean to say that the container cannot mount new devices in the `/dev` directory space? or something else - I'm having trouble understanding what this sentence is trying to say